cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/keyspaces/KeyspaceCreator.java

```diff
@@ -118,8 +118,7 @@ private void awaitKeyspaceReady() throws CradleStorageException
 		
 		if (getKeyspaceMetadata() == null)
 			throw new CradleStorageException(
-					"Keyspace '" + keyspace + "' unavailable after " + attempt * keyspaceWaitTimeout +
-							"ms of awaiting");
+					"Keyspace '" + keyspace + "' unavailable after " + settings.getTimeout() + "ms of awaiting");
```

Comment:
When `settings.getTimeout()` is less then `KEYSPACE_WAIT_TIMEOUT` you will wait `KEYSPACE_WAIT_TIMEOUT` ms, not `settings.getTimeout()` ms as you report in this message